### PR TITLE
Add further details to the help window

### LINKED ISSUE #25: Onboarding
As a user, I want to be able to understand how the app works from start to end so that I can be able to provide the necessary inputs to perform a particular action on the app.

### PR DESCRIPTION
**Pull request by howardwhw2**
The help window currently only links to user guide which is lengthy and may turn off new users from picking up the app.

Adding summarised portions of the user guide would increase the readability

As a step to improving the new user experience, summarised part os the user guide is added to the help window to make learning how to use the app easier

To do:
- [x] Design
- [x] Integration
- [ ] Documentation
- [ ] Additional details (when features are finalised)

Note: No testing was added since this is purely a UI increment 

closes  #25

### CODE REVIEW

**Comment by waffledood** [comment]
Hi Howard, do add the line "closes #id" (where id is the issue number of your feature) in your PR description, so that upon merging your PR, the issue corresponding to your feature gets closed as well

File: src/main/resources/view/HelpWindow.fxml [hunk 1]
```diff
@@ -1,44 +1,51 @@
...
 <?import javafx.scene.control.Label?>
+<?import javafx.scene.control.SplitPane?>
 <?import javafx.scene.image.Image?>
-<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.AnchorPane?>
+<?import javafx.scene.layout.StackPane?>
 <?import javafx.stage.Stage?>
 
-<fx:root resizable="false" title="Help" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root resizable="false" title="Help" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (waffledood) [inline src/main/resources/view/HelpWindow.fxml:13]: I realized you're using JavaFX 17? Did you have to import anything or update the gradle build script? If not, then it should be fine

**Review by waffledood — APPROVED** [review]
LGTM! Just a query with regards to the JavaFX version

File: src/main/resources/view/HelpWindow.fxml (hunk 1 above)
Comment (howardwhw2) [reply src/main/resources/view/HelpWindow.fxml]: I have changed it to version 11. Thanks for spotting that issue

**Pull request merged**